content/blog/2019-02-20-react-translations.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "¡React en español! And Other Languages"
```

Comment:
Why are we singling out Spanish? Both in this title and in the thanks at the end. It sounds like we have three languages available already and more soon right?

I guess from https://isreacttranslatedyet.com/ Spanish is the first to 100%? I might say "Special congrats to the Spanish translation team for being the first to hit 100% completion." or similar in the body of the post, but I worry about making the title seem too narrow – people might think it's just Spanish.

(Also, to my eyes this would look better with a lowercase "and".)

Reply:
Correct, they're the first to actually "finish" finish. I don't really know what's the best way to write it without implying the other two were not as good.

Reply:
@sophiebits Do you have ideas for alternative titles?

Reply:
maybe…

> React: Now en español, 日本語で, em português, and More

/shrug

(also please get a japanese speaker to check that if you wanna go with that)

Reply:
How about:

> React in español, portuguese do brazíl, 日本語, and more!

Reply:
Another title idea:

> Is React Translated Yet? Sí, はい, sim!